math/catalan/catalannumber.go

```diff
@@ -0,0 +1,16 @@
+// catalannumber.go
```

Comment:
Add a small explanation of what is a Catalan number.

Reply:
Done